defaults/main.yml

```diff
@@ -23,6 +23,9 @@ aiida_postgres_db_pw: "ne9N_LDK-*JSS"
 aiida_postgres_db_user: aiida
 aiida_profile_name: generic
 
+# also install and configure the components specified here
+aiida_components:
```

Comment:
I would probably put the full list by default (?)

In any case, empty list is `[]`

Reply:
Default behaviour is now per requested spec. Fixed.

Reply:
Thanks a lot!